product_docs/docs/biganimal/release/administering_cluster/01_portal_access.mdx

```diff
@@ -5,129 +5,91 @@ redirects:
     - 01_user_access
 ---
 
+BigAnimal uses role-based access controls to grant users access to different parts of the application using organization-level and project-level roles. Roles are sets of permissions. Roles are used to manage permissions assigned to users.
 
-BigAnimal uses role-based access controls to grant users access to different parts of the application.
+Each customer has a unique BigAnimal organization. Each organization has at least one project by default. An organization's database clusters are deployed and managed in the customer's cloud account within a project. Multiple projects can be created within a single organization.
 
-## Organizations
+With multiple projects within an organization you can: 
+- separate workflows to provide secure and isolated environments 
+- assign different users to different projects or give different roles to users in different projects to ensure they have the correct level of permissions
 
-Each subscribed customer has a unique *organization*. Each organization’s database clusters are deployed and managed in the customer's cloud account. This approach ensures complete segregation of customer data between organizations.
+In order to access a BigAnimal organization, each user needs to be first added to the organization's identity provider. For more informations, ee [Setting up your identity provider](/biganimal/latest/getting_started/identity_provider/). Your identity provider establishes the identity of users that can log in to BigAnimal. 
```

Comment:
`ee [Setting up your identity provider]`, `ee` should be `see`?

Reply:
fixed